client.js

```diff
@@ -121,27 +121,36 @@ RingpopClient.prototype._request = function _request(opts, endpoint, head, body,
         return;
     }
 
-    this.subChannel.waitForIdentified({
-        host: opts.host
-    }, function onIdentified(err) {
-        if (err) {
-            callback(err);
-            return;
-        }
+    try {
+        request();
+    } catch(e) {
+        callback(e);
+        return;
+    }
```

Comment:
Do not know if try catching every request is a good idea, we could also only try catch only pings and ping-reqs for example